packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilterDialog.tsx

```diff
@@ -20,14 +20,33 @@ import { PresentationInstanceFilterInfo } from "./PresentationFilterBuilder";
 import { PresentationInstanceFilter } from "./PresentationInstanceFilter";
 import { filterRuleValidator, isFilterNonEmpty } from "./Utils";
 
+/**
+ * Data structure that describes source to gather properties from.
+ * @beta
+ */
+export interface PropertiesSource {
```

Comment:
Maybe `PresentationInstanceFilterPropertiesSource`? `PropertiesSource` sounds very generic and in a package, containing many different components, makes it unclear what component it's related to.

Reply:
Renamed